### PR TITLE
test(language-service): add missing tests for templateUrls

### PR DESCRIPTION
**Pull request by ayazhafiz**
Add missing tests for duplicate, missing template diagnostics

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by kyliau — CHANGES_REQUESTED** [review]

File: packages/language-service/test/diagnostics_spec.ts [hunk 1]
```diff
@@ -475,6 +475,59 @@ describe('diagnostics', () => {
             `Module '"../node_modules/@angular/core/core"' has no exported member 'OpaqueToken'.`);
   });
 
+  describe('templates', () => {
+    it('should report diagnostic for missing template or templateUrl', () => {
+      const fileName = mockHost.addCode(`
+        import { Component, NgModule } from '@angular/core';
```
[the remark this answers is not in the document]
Comment (ayazhafiz) [reply packages/language-service/test/diagnostics_spec.ts]: Fixed.

**Review by kyliau — APPROVED** [review]

**Comment by ayazhafiz** [comment]
Caretaker: please restart build.

**Comment by AndrewKushnir** [comment]
@ayazhafiz it looks like you need to rebase this PR on top of the latest master, here is an error message from CI:
```
CONFLICT (content): Merge conflict in packages/language-service/test/diagnostics_spec.ts\nPatch failed at 0001 test(language-service): add missing tests for templateUrls
The copy of the patch that failed is found in: .git/rebase-apply/patch
When you have resolved this problem, run "git rebase --continue".
If you prefer to skip this patch, run "git rebase --skip" instead.
To check out the original branch and stop rebasing, run "git rebase --abort
```
It also looks like the most recent commit will fail the lint check due to commit description. Could you please squash commits as well?
Thank you.

**Comment by AndrewKushnir** [comment]
@ayazhafiz thanks for rebasing this PR, CI is "green" now. Feel free to put "merge" label back if/when PR is ready to be merged. Thank you.
